crates/language/src/target_language.rs

```diff
@@ -371,6 +374,61 @@ impl PatternLanguage {
     }
 }
 
+impl ValueEnum for PatternLanguage {
```

Comment:
I think you could just derive this. See this example: https://github.com/clap-rs/clap/issues/4327